docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,628 @@
+
+At this point, you are ready to start generating UID2 tokens using the SDK.
+
+### Using the UID2 Integration Environment
+
+By default, the SDK is configured to work with the UID2 production environment https://prod.uidapi.com . If you want to use the UID2 integration environment instead, provide the following URL in your call to initialize UID2Manager:
+
+#### For Android:
+
+```
+UID2Manager.init(
+
+  context = this,
+  serverUrl = "https://operator-integ.uidapi.com"
+)
+```
+
+#### For iOS:
+
+```
+// Must be set before UID2Manager.shared is accessed
+
+UID2Settings.shared.environment = .custom(
+
+  url: URL(string: "https://operator-integ.uidapi.com")!
+
+)
+```
+
+:::note
+Tokens from the UID2 integration environment are not valid for passing to the bid stream. For the integration environment, you will have different subscription ID and public key values.
+:::
+
+### Optional: Reduce Latency by Setting the API Base URL for the Production Environment
+
+By default, in the production environment, the UID2 Mobile SDKs make API calls to a UID2 server in the USA. Depending on where your users are based, you might consider choosing a server closer to your users to reduce latency.
+
+For example, a publisher in Singapore can set the server URL to https://sg.prod.uidapi.com . This is still the UID2 production environment, but the servers are in Singapore.
+
+For the list of possible base URLs, see [Environments](../getting-started/gs-environments.md).
+
+A publisher can also set the base URL to [https://global.prod.uidapi.com](https://global.prod.uidapi.com). This URL directs readers to a region geographically close to them, which is ideal if your audience is geographically distributed.
+
+To specify a different UID2 server, you can change it in the init call, as shown in the following examples.
+
+#### For Android:
+ 
+```
+UID2Manager.init(
+  context = this,
+  serverUrl = " https://global.prod.uidapi.com"
+)
+```
+
+#### For iOS:
+
+```
+// Must be set before UID2Manager.shared is accessed
+UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://global.prod.uidapi.com")!
+
+)
+
+// or use a named environment
+UID2Settings.shared.environment = .sydney
+```
+
+## Configure the UID2 Mobile SDK
+
+UID2 provides the publisher with the following values required to generate the UID2 token on the client side:
+
+- A subscription ID
+- A public key
+
+You'll have one set of these values for your publisher testing environment, and a separate set for your production environment.
+
+To configure the SDK, you are required to pass in the public key and subscription ID that you received during account setup, as well as the user’s hashed or unhashed directly identifying information ([DII](../ref-info/glossary-uid.md#gl-dii)) (email address or phone number) into (**GWH__SW this sentence just stops not sure how to connect it with the next bit.**)
```

Comment:
it continues onto the next couple lines of codes. Just overall just meant 

_pass the subscription ID, public key and DII  into Android's UID2Manager.getInstance().generateIdentity() method or iOS's UID2Manager.shared.generateIdentity() method_

Reply:
Updated, thx.